features/step_definitions/helper_services.rb

```diff
@@ -471,20 +471,22 @@
     @result = admin.cli_exec(:delete, n: _project.name, object_type: "pod", object_name_or_id: _pod.name)
     raise "could not delete broken iSCSI pod" unless @result[:success]
   else
-    env.node_hosts.each do |host|
-      setup_commands = [
-        "echo 'InitiatorName=iqn.2016-04.test.com:test.img' > /etc/iscsi/initiatorname.iscsi",
-        "sed -i '/^node.session.auth./'d  /etc/iscsi/iscsid.conf",
-        "cat >> /etc/iscsi/iscsid.conf << EOF\n" +
-          "node.session.auth.authmethod = CHAP\n" +
-          "node.session.auth.username = 5f84cec2\n" +
-          "node.session.auth.password = b0d324e9\n" +
-          "EOF\n",
-        "systemctl enable iscsid",
-        "systemctl restart iscsid"
-      ]
-      res = host.exec_admin(*setup_commands)
-      raise "iSCSI initiator setup commands error" unless res[:success]
+    if env.version_cmp("4.0", user: user) < 0
```

Comment:
As Peter said, master branch is only for 4.x.

Just FYI, in version comparisons better to use `env.version_gt/ge/lt/le/eq` methods instead of `env.version_cmp`. They are more easily readable what they mean. Now it is hard to tell whether condition is for 4.0+ of 3.x